Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Tools/Error.hs

```diff
@@ -212,6 +214,7 @@ instance IsHTTPError DriverQuoteError where
   toErrorCode = \case
     FoundActiveQuotes -> "FOUND_ACTIVE_QUOTES"
     DriverOnRide -> "DRIVER_ON_RIDE"
+    DriverOnRideButNoRideWithRideStatusInprogress -> "Driver_On_Ride_But_No_Ride_With_Ride_Status_INPROGRESS"
```

Comment:
can you capitalise this ? maybe a bit shorter also i.e. `DriverOnRideButNoInprogressRide` 
 `DRIVER_ON_RIDE_BUT_NO_INPROGRESS_RIDE`